src/service/video-manager-impl.js

```diff
@@ -172,6 +174,40 @@ export class VideoManager {
     }
   }
 
+
+  /**
+   * Install the necessary listeners to be notified when the user changes
+   * the orientation of their device
+   *
+   * @param {VideoEntry} entry
+   * @private
+   */
+  maybeInstallOrientationObserver_(entry) {
+    // TODO(@wassgha): Remove this later. For now, the orientation observer
+    // is only useful for automatically putting videos in fullscreen.
+    if (!entry.video.element.hasAttribute(VideoAttributes.AUTOFULLSCREEN)) {
+      return;
+    }
+
+    // TODO(@wassgha) Check support status for orientation API and updateVisibility
+    // this as needed.
+    //
+    // Chrome apparently considers 'orientationchange' to be an untrusted
+    // event, while 'change' on screen.orientation is considered a user
+    // interaction
+    if (screen.orientation !== undefined) {
```

Comment:
note sue if `screen` is available in all browsers (e.g. IE11) also we want to get the `screen` from the current `win` not the global `screen` so let's do:
```
const screen = this.ampdoc_.win.screen;
if (screen && screen.orientation) 
```